manifests/install.pp

```diff
@@ -28,14 +31,18 @@
   file { $teleport::extract_path:
     ensure => $directory_ensure,
   } ->
-  archive { $teleport::archive_path:
-    ensure        => $ensure,
-    extract       => true,
-    extract_path  => $teleport::extract_path,
-    source        => $teleport::archive_url,
-    creates       => "${teleport::extract_path}/teleport",
-    checksum      => $teleport::checksum,
-    checksum_type => $teleport::checksum_type,
+  exec { 'download-teleport-latest':
+    command => "/usr/bin/aws s3 --region=${::region} cp ${teleport::archive_url} ${teleport_latest_path}",
```

Comment:
Is there anything that ensures the AWS cli is available at this location on machines that install teleport?

It definitely won't work for CoreOS, but that's fine because we don't need to distribute Teleport through Puppet on that.

Reply:
It's used by various other modules. I think the base module installs it